riak_test/yz_startup_shutdown.erl

```diff
@@ -134,3 +141,29 @@ verify_drain_called(Cluster) ->
                              ok =:= Result
                      end,
                      Results).
+
+%% @private
+%% @doc For each node in `Cluster', verify that the ibrowse configuration has
+%% been applied.
+verify_ibrowse_config([Node1|_] = Cluster) ->
+    {ResL, []} = rpc:multicall(Cluster, yz_solr, get_ibrowse_config, []),
+    lists:foreach(
+      fun(Config) ->
+              MaxSessions = proplists:get_value(max_sessions, Config),
+              MaxPipelineSize = proplists:get_value(max_pipeline_size, Config),
```

Comment:
You may want to macro-tize these names like we did in https://github.com/basho/yokozuna/pull/615, if we ever change them going forward, life is easier.